CMakeLists.txt

```diff
@@ -1,15 +1,15 @@
 cmake_minimum_required(VERSION 2.8)
 
 # Change this while bumping up the version, needed for RHEL package
-set(VERSION "1.4.0")
+set(VERSION "1.4.1")
```

Comment:
why not keep this as 1.4.0? We havent released this yet right?

Reply:
Some customers have taken the code to try out streaming and sync feature and there version is already 1.4.0. Later it may create confusion on whether its official build or the local build. Just to avoid that confusion later while debugging issues bumping up this version to 1.4.1